artio-codecs/src/main/resources/uk/co/real_logic/artio/messages/message-schema.xml

```diff
@@ -499,6 +499,9 @@
                  description="notifies library instances that they have been timed out, added for monitoring purposes">
         <field name="libraryId" id="1" type="LibraryId"/>
         <field name="connectCorrelationId" id="2" type="CorrelationId"/>
+        <group name="sessions" id="76" dimensionType="groupSizeEncoding">
```

Comment:
You should increment the version and declare since which this group is present. Also any particular reason for those IDs?

Reply:
Updated. Thanks.